browser.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-env browser */
 
-const isBlinkBasedBrowser = /\b(Chrome|Chromium)\//.test(navigator.userAgent);
+const isBlinkBasedBrowser = typeof navigator !== 'undefined' ? /\b(Chrome|Chromium)\//.test(navigator.userAgent): false;
```

Comment:
```suggestion
const isBlinkBasedBrowser = typeof navigator !== 'undefined' && /\b(Chrome|Chromium)\//.test(navigator.userAgent);
```

Reply:
The RHS of the `:` in a ternary is implicitly the false case, so this can be simplified as such - namely that the short-circuit will result in a `false` and `RegExp.prototype.text()` returns a boolean.